src/frontends/tensorflow_common/src/op/ctc_loss.cpp

```diff
@@ -33,33 +33,33 @@ OutputVector translate_ctc_loss_op(const NodeContext& node) {
         // since OpenVINO CTCLoss accepts only batch-major logist
         // we need to transpose it into [batch_size, time_size, num_classes] format
         // from [time_size, batch_size, num_classes]
-        ov::AxisVector logits_order = {1, 0, 2};
-        logits = ov::frontend::tensorflow::make_transpose(logits, logits_order);
+        AxisVector logits_order = {1, 0, 2};
+        logits = tensorflow::make_transpose(logits, logits_order);
     }
 
     // Transform decoded labels from the sparse format into dense format
     // Convert to the signed type since the mask with minus one is formed below
-    decoded_values = make_shared<Convert>(decoded_values, ov::element::i64);
+    decoded_values = make_shared<Convert>(decoded_values, element::i64);
     // OpenVINO ScatterND operation requires indices to be signed
-    decoded_indices = make_shared<Convert>(decoded_indices, ov::element::i64);
+    decoded_indices = make_shared<Convert>(decoded_indices, element::i64);
     // OpenVINO CTCLoss requires logit_length to be signed
-    logit_length = make_shared<Convert>(logit_length, ov::element::i64);
+    logit_length = make_shared<Convert>(logit_length, element::i64);
 
-    auto logits_shape = make_shared<ShapeOf>(logits, ov::element::i64);
+    auto logits_shape = make_shared<ShapeOf>(logits, element::i64);
     auto dense_shape = make_shared<Slice>(logits_shape,
-                                          make_shared<Constant>(ov::element::i64, ov::Shape{1}, 0),
-                                          make_shared<Constant>(ov::element::i64, ov::Shape{1}, 2),
-                                          make_shared<Constant>(ov::element::i64, ov::Shape{1}, 1));
-    auto minus_one_value = make_shared<Constant>(decoded_values.get_element_type(), ov::Shape{}, -1);
+                                          make_shared<Constant>(element::i64, Shape{}, 0),
+                                          make_shared<Constant>(element::i64, Shape{}, 2),
+                                          make_shared<Constant>(element::i64, Shape{}, 1));
+    auto minus_one_value = make_shared<Constant>(element::i64, Shape{}, -1);
```

Comment:
Sounds like const_minus_one

Reply:
Please, check variables naming, why everywhere they are different? const_zero, zero, zero_element, minus_one, const_minus_one, minus_one_const, etc...

Reply:
It makes sense to have the common names for such constant. Let us do it in the separate PR. It is a sort of improvement of our code and not blocker.